bin/single-new-eks-graviton-opensource-observability.ts

```diff
@@ -1,5 +1,5 @@
 import { configureApp } from '../lib/common/construct-utils';
-import SingleNewEksGravitonOpenSourceObservabilityConstruct from '../lib/single-new-eks-opensource-observability-construct/graviton-index';
+import SingleNewEksGravitonOpenSourceObservabilityConstruct from '../lib/single-new-eks-opensource-observability/graviton-index';
```

Comment:
Why is class name `SingleNewEksGravitonOpenSourceObservabilityConstruct ` still having Construct ? We want to totally get rid of word Construct with Pattern. Also we dont want -construct to be removed. We would rather want it to be renamed to -pattern.